### PR TITLE
multi_disk_random_hotplug: Add interval time for unplugging parallel

### PR DESCRIPTION
**Pull request by YongxueHong**
Need some interval time for unplugging parallel way in windows.

ID: 1866579

### CODE REVIEW

**Comment by YongxueHong** [comment]
@qingwangrh  please help me to review, thanks.
BTW please run other sub-cases in order to no regression errors.
Thanks.

**Comment by qingwangrh** [comment]
I did small change to improve test efficiency:

--- a/qemu/tests/cfg/multi_disk_random_hotplug.cfg
+++ b/qemu/tests/cfg/multi_disk_random_hotplug.cfg
@@ -9,7 +9,7 @@
     stg_image_name = "images/stg%s"
     stg_image_num = 20
     stg_image_size = 128M
-    repeat_times = 3
+    repeat_times = 2
     wait_between_hotplugs = 2
     wait_after_hotplug = 10
     wait_between_unplugs = 2
@@ -80,6 +80,8 @@
     variants:
         - @serial:
         - parallel:
+            Windows:
+                interval_time_unplug = 60
             multi_disk_type = parallel
             monitors += " TestQMP1 TestQMP2 TestQMP3 TestQMP4"
             monitor_type_TestQMP1 = qmp
@@ -92,11 +94,14 @@
                 acquire_lock_timeout = 7200
                 verify_unplug_timeout = 300
             parallel.single_type:
+                Linux:
+                    stg_image_num = 64
                 virtio_scsi:
-                    no Windows
-                    monitors += " TestQMP5 TestQMP6 TestQMP7 TestQMP8 TestQMP9 TestQMP10"
-                    monitors += " TestQMP11 TestQMP12 TestQMP13 TestQMP15 TestQMP15 TestQMP16"
-                    monitors += " TestQMP17 TestQMP18 TestQMP19 TestQMP20"
+                    #no Windows
+                    monitors += " TestQMP5 TestQMP6 TestQMP7 TestQMP8"
+                    #monitors += " TestQMP5 TestQMP6 TestQMP7 TestQMP8 TestQMP9 TestQMP10"
+                    #monitors += " TestQMP11 TestQMP12 TestQMP13 TestQMP15 TestQMP15 TestQMP16"
+                    #monitors += " TestQMP17 TestQMP18 TestQMP19 TestQMP20"
                     monitor_type_TestQMP5 = qmp
                     monitor_type_TestQMP6 = qmp
                     monitor_type_TestQMP7 = qmp


python ConfigTest.py --testcase=multi_disk_random_hotplug --iothread_scheme=roundrobin --nr_iothreads=2 --platform=x86_64 --guestname=Win2019,RHEL.8.3.0 --driveformat=virtio_scsi,virtio_blk --nicmodel=virtio_net --imageformat=qcow2 --machines=i440fx --customsparams="qemu_force_use_drive_expression = no\nimage_aio=native\ncd_format=ide" 

(01/22) Host_RHEL.m8.u3.product_av.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.8.3.0.x86_64.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.default_install.aio_threads: PASS (911.88 s)
 (02/22) Host_RHEL.m8.u3.product_av.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.8.3.0.x86_64.io-github-autotest-qemu.rh_kernel_update: PASS (224.51 s)
 (03/22) Host_RHEL.m8.u3.product_av.qcow2.virtio_blk.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.default_install.aio_threads: PASS (2756.82 s)
 (04/22) Host_RHEL.m8.u3.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.3.0.x86_64.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.default_install.aio_threads: PASS (887.20 s)
 (05/22) Host_RHEL.m8.u3.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.3.0.x86_64.io-github-autotest-qemu.rh_kernel_update: PASS (275.18 s)
 (06/22) Host_RHEL.m8.u3.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.default_install.aio_threads: PASS (1879.63 s)
 (07/22) Host_RHEL.m8.u3.product_av.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.8.3.0.x86_64.io-github-autotest-qemu.multi_disk_random_hotplug.all_types: PASS (283.34 s)
 (08/22) Host_RHEL.m8.u3.product_av.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.8.3.0.x86_64.io-github-autotest-qemu.multi_disk_random_hotplug.single_type: PASS (182.61 s)
 (09/22) Host_RHEL.m8.u3.product_av.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.8.3.0.x86_64.io-github-autotest-qemu.multi_disk_random_hotplug.parallel.all_types: PASS (222.17 s)
 (10/22) Host_RHEL.m8.u3.product_av.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.8.3.0.x86_64.io-github-autotest-qemu.multi_disk_random_hotplug.parallel.single_type: ERROR: Failed to insert device:\ndevice\n  aid = None\n  aobject = stg40\n  parent_bus = ({'busid': 'drive_stg40'}, {'aobject': 'pci.0'}, {'busid': 'iothread_bus_iothread1'})\n  child_bus = []\n  params:\n    driver = virtio-blk-pci\n    id = stg40\n    drive = drive_stg... (57.66 s)
 (11/22) Host_RHEL.m8.u3.product_av.qcow2.virtio_blk.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.multi_disk_random_hotplug.all_types: PASS (1767.90 s)
 (12/22) Host_RHEL.m8.u3.product_av.qcow2.virtio_blk.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.multi_disk_random_hotplug.single_type: PASS (1058.54 s)
 (13/22) Host_RHEL.m8.u3.product_av.qcow2.virtio_blk.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.multi_disk_random_hotplug.parallel.all_types: ERROR: unplug--> Actual: 19 disks. Expected: 20 disks. (1045.80 s)
 (14/22) Host_RHEL.m8.u3.product_av.qcow2.virtio_blk.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.multi_disk_random_hotplug.parallel.single_type: PASS (967.44 s)
 (15/22) Host_RHEL.m8.u3.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.3.0.x86_64.io-github-autotest-qemu.multi_disk_random_hotplug.all_types: PASS (317.99 s)
 (16/22) Host_RHEL.m8.u3.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.3.0.x86_64.io-github-autotest-qemu.multi_disk_random_hotplug.single_type: PASS (1072.02 s)
 (17/22) Host_RHEL.m8.u3.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.3.0.x86_64.io-github-autotest-qemu.multi_disk_random_hotplug.parallel.all_types: PASS (219.59 s)
 (18/22) Host_RHEL.m8.u3.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.3.0.x86_64.io-github-autotest-qemu.multi_disk_random_hotplug.parallel.single_type: ERROR: unplug--> Actual: 0 disks. Expected: 64 disks. (1976.02 s)
 (19/22) Host_RHEL.m8.u3.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.multi_disk_random_hotplug.all_types: PASS (1768.40 s)
 (20/22) Host_RHEL.m8.u3.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.multi_disk_random_hotplug.single_type: PASS (1000.82 s)
 (21/22) Host_RHEL.m8.u3.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.multi_disk_random_hotplug.parallel.all_types:PASS (1541.77 s)
 (22/22) Host_RHEL.m8.u3.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.multi_disk_random_hotplug.parallel.single_type: PASS (988.67 s)


http://fileshare.englab.nay.redhat.com/pub/section2/images_backup/qbugs/code2352/2020-08-10-1906


Please help the check the failed case.  it seems like the message is wrong due to the action should be plug.

File: qemu/tests/cfg/multi_disk_random_hotplug.cfg [hunk 1]
```diff
@@ -81,6 +81,8 @@
     variants:
         - @serial:
         - parallel:
+            Windows:
+                interval_time_unplug = 60
```
Comment (qingwangrh) [inline qemu/tests/cfg/multi_disk_random_hotplug.cfg:85]: This issue related to scsi, suggest adding scsi filter in case suppress some product bug.

Comment (YongxueHong) [reply qemu/tests/cfg/multi_disk_random_hotplug.cfg:85]: you mean just run `parallel` case on virtio-scsi? no need virtio-blk?

Comment (YongxueHong) [reply qemu/tests/cfg/multi_disk_random_hotplug.cfg:85]: sorry, I got you, will update.

Comment (YongxueHong) [reply qemu/tests/cfg/multi_disk_random_hotplug.cfg]: Hi @qingwangrh 
thanks, have updated.

**Comment by qingwangrh** [comment]
python ConfigTest.py --testcase=multi_disk_random_hotplug --iothread_scheme=roundrobin --nr_iothreads=2 --platform=x86_64 --guestname=Win2019,RHEL.8.4.0 --driveformat=virtio_scsi,virtio_blk --nicmodel=virtio_net --imageformat=qcow2 --machines=i440fx --customsparams="qemu_force_use_drive_expression = no\nimage_aio=native\ncd_format=ide" --clone=no
(01/15) Host_RHEL.m8.u3.product_av.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.8.4.0.x86_64.io-github-autotest-qemu.multi_disk_random_hotplug.all_types: PASS (308.56 s)
 (02/15) Host_RHEL.m8.u3.product_av.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.8.4.0.x86_64.io-github-autotest-qemu.multi_disk_random_hotplug.single_type: PASS (211.74 s)
 (03/15) Host_RHEL.m8.u3.product_av.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.8.4.0.x86_64.io-github-autotest-qemu.multi_disk_random_hotplug.parallel.all_types: PASS (327.54 s)
 (04/15) Host_RHEL.m8.u3.product_av.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.8.4.0.x86_64.io-github-autotest-qemu.multi_disk_random_hotplug.parallel.single_type: PASS (224.45 s)
 (05/15) Host_RHEL.m8.u3.product_av.qcow2.virtio_blk.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.multi_disk_random_hotplug.all_types: PASS (2432.83 s)
 (06/15) Host_RHEL.m8.u3.product_av.qcow2.virtio_blk.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.multi_disk_random_hotplug.single_type: PASS (1422.45 s)
 (07/15) Host_RHEL.m8.u3.product_av.qcow2.virtio_blk.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.multi_disk_random_hotplug.parallel.all_types: PASS (2110.43 s)
 (08/15) Host_RHEL.m8.u3.product_av.qcow2.virtio_blk.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.multi_disk_random_hotplug.parallel.single_type: PASS (1407.86 s)
 (09/15) Host_RHEL.m8.u3.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.x86_64.io-github-autotest-qemu.multi_disk_random_hotplug.all_types: PASS (313.80 s)
 (10/15) Host_RHEL.m8.u3.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.x86_64.io-github-autotest-qemu.multi_disk_random_hotplug.single_type: PASS (1388.60 s)
 (11/15) Host_RHEL.m8.u3.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.x86_64.io-github-autotest-qemu.multi_disk_random_hotplug.parallel.all_types: PASS (321.84 s)
 (12/15) Host_RHEL.m8.u3.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.x86_64.io-github-autotest-qemu.multi_disk_random_hotplug.parallel.single_type: PASS (1469.68 s)
 (13/15) Host_RHEL.m8.u3.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.multi_disk_random_hotplug.all_types: PASS (2376.69 s)
 (14/15) Host_RHEL.m8.u3.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.multi_disk_random_hotplug.single_type: PASS (1351.96 s)
 (15/15) Host_RHEL.m8.u3.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.Win2019.x86_64.io-github-autotest-qemu.multi_disk_random_hotplug.parallel.all_types: PASS (2376.85 s)

LGTM

**Pull request merged**